src/game/scenes/lobby.c

```diff
@@ -1090,6 +1090,8 @@ void lobby_tick(scene *scene, int paused) {
 
                             game_player_set_selectable(challengee, 1);
 
+                            chr_score_reset(game_player_get_score(game_state_get_player(gs, 0)), true);
```

Comment:
There's a third place a game can launch, but I'd still recommend putting these in lobby_create for simplicity.